crates/holochain/src/conductor/cell.rs

```diff
@@ -591,6 +591,7 @@ impl holochain_p2p::event::HcP2pHandler for Cell {
         Box::pin(async {
             #[cfg(not(feature = "unstable-countersigning"))]
             {
+                drop(message);
```

Comment:
I'm using some of my credit for asking stupid questions: Why is this drop needed here?

Reply:
To silence the compiler warning about unused parameter : )